tensorflow/python/ops/rnn_cell_impl.py

```diff
@@ -556,9 +556,9 @@ def call(self, inputs, state):
     Args:
       inputs: `2-D` tensor with shape `[batch_size, input_size]`.
       state: An `LSTMStateTuple` of state tensors, each shaped
-        `[batch_size, self.state_size]`, if `state_is_tuple` has been set to
```

Comment:
```self.state_size``` is an ```LSTMStateTuple``` if ```state_is_tuple``` is ```True```, so here should be ```self._num_units```.

Reply:
replace `self._num_units` with just `num_units`.

Reply:
Done. Thanks.

Reply:
@ebrevdo Done. Thanks.